function prueba_callback()
    try 
        disp(['Prueba de funci√≥n callback con timer.']);
    catch ME
        disp('Error.');
        disp(ME.message);
    end 
end